apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -50,17 +46,27 @@ export type HomepageProps = {
   };
 };
 
-type StaticHeader = typeof translations.header;
 type StaticHomepage = typeof translations.homepage;
 
 export const makeHomepageProps = (
   strapiHomepage: StrapiHomepage,
-  staticHeader: StaticHeader,
   staticHomepage: StaticHomepage
 ): HomepageProps => ({
-  ...makeHomepagePropsFromStatic(staticHeader, staticHomepage),
+  ...makeHomepagePropsFromStatic(staticHomepage),
   comingsoonDocumentation:
     strapiHomepage.data.attributes.comingsoonDocumentation,
+  hero: strapiHomepage.data.attributes.heroSlider.map((slide) => ({
+    title: slide.title,
+    titleColor: slide.titleColor,
+    backgroundImage: slide.backgroundImage?.data.attributes,
+    callToAction: slide.callToAction && {
+      link: {
+        href: slide.callToAction.link.href,
+        text: slide.callToAction.link.text,
+        target: slide.callToAction.link.target,
+      },
```

Comment:
is this needed?

Reply:
Yes, as `slide.callToAction` could be `null` or `undefined`